apng/__init__.py

```diff
@@ -458,7 +459,7 @@ def from_bytes(cls, b):
 					# IDAT inside chunk, go to next frame
 					frame_chunks.append(end)
 					frames.append((PNG.from_chunks(frame_chunks), control))
-					
+					new_frame = True
```

Comment:
This line should be inserted after L467 and changed to `frame_has_head_chunks = False`.